ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -1277,12 +1278,17 @@ - (NSDate *)pickerDefaultDate {
     return ORKTimeOfDayDateFromComponents(newDateComponents);
 }
 
+- (NSInteger)pickerMinuteInterval {
```

Comment:
is this method necessary? I don't see it being referenced anywhere..

Reply:
Thanks for catching this! I added them at the first place, but seems they are extraneous now. Having the `minuteInterval` instance variable alone should do. Removed the getters.